base-server.js

```diff
@@ -281,6 +282,37 @@ BaseServer.prototype = {
   },
 
   /**
+   * Load options from both CLI and environment
+   *
+   * @param {object} process Current process.
+   * @param {object} defaults Default options.
+   * @return {object} options.
+   *
+   * @example
+   * options = app.loadOptions(process, defaults)
+   *
+   */
+  loadOptions: function loadOptions (process, defaults) {
+    program
+      .option('-p, --port [port]', 'Set up a port')
+      .option('-h, --host [host]', 'Set up a host')
+      .option('-c, --cert [cert]', 'Set up a cert')
```

Comment:
cert → SSL certificate. But add `mostly file with .cert extentions` to help users.